docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,680 @@
+. The collection requires the following two variables to work:
+_API_BASE_HOST_:: The base host for the API. This is the base URL for the API. For example, https://api.openshift.com.
+_SSO_BASE_HOST_:: The base host for the SSO. This is the base URL for the SSO. For example, https://sso.redhat.com/auth/realms/redhat-external.
+You can define the environment variables shown in an environment variable (`.env`) file and save the file in the root directory. If you don't explicitly define these environment variables, the collection uses the URLs shown in the examples.
+. Open a terminal window.
+. Run a module in the terminal to perform a single task. For example, the following command shows how to run a module that creates a Kafka instance. Replace _<OFFLINE_TOKEN>_ with your OpenShift offline token.
++
+.Example create_kafka module
+[source,shell]
+----
+$ ansible localhost -m rhoas.rhoas.create_kafka -a 'name=unique-kafka-name billing_model=standard cloud_provider=aws plan="developer.x1" region="us-east-1" openshift_offline_token=<OFFLINE_TOKEN>'
+----
+
++
+. Ansible runs the `rhoas.rhoas.create_kafka` task in the terminal and creates the instance.
+. Next, you will create a playbook, based on an existing example. Navigate to the `rhoas` folder in the `.ansible` directory.
++
+[source]
+----
+$ cd rhoas
+----
+
+. Open the `rhoas_test` example playbook. You can also open a web browser and navigate to https://github.com/redhat-developer/app-services-ansible/blob/main/rhoas_test.yml to open the `rhoas_test` playbook.
+. Inspect the contents of the example playbook. In particular, observe that the playbook has modules for certain tasks. You will use these modules, which are also shown below, to create a new playbook which will complete the following tasks:
++
+* Create and delete a Kafka instance
+* Create and delete a service account
+* Create an Access Control List (ACL) permission bindings
+* Create, update, and delete a topic
++
+[NOTE]
+The playbook uses your offline token to authenticate with the Kafka Management API. If the token is not passed in as an argument per task, the module attempts to read it from the _`OFFLINE_TOKEN`_ environment variable.
++
+[NOTE]
+====
+The example playbook used in this section includes comments that indicate how to directly specify values rather than fetching them dynamically. For example, to specify a Kafka instance ID, a comment in the playbook states that you can include the following line:
+
+[source, subs="+quotes"]
+----
+kafka_id: __<kafka_id>__
+----
+====
++
+. Create a new playbook file and save it as `rhoas_kafka.yml` in the `rhoas` directory.
+. Open the `rhoas_kafka.yml` file in your IDE.
+. In the top section of the new playbook, you define the group of hosts on which to run the playbook and its name. In the following example, the host is a local host and the connection is local because you are in a virtual environment. Copy the top section example and paste it into the `rhoas_kafka.yml` file.
```

Comment:
Suggest: "In the top section of the new playbook, you define the name of the playbook and the group of hosts on which to run it."